src/nostr-zap/zap-utils.ts

```
@@ -0,0 +1,207 @@
+  const event = nip57.makeZapRequest({
+    profile,
+    event:
+      nip19Target && nip19Target.startsWith('note')
+        ? decodeNip19Entity(nip19Target)
+        : undefined,
+    amount,
+    relays,
+    comment: comment || '',
+  });
+
+  if (nip19Target && nip19Target.startsWith('naddr')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)